### ISSUE TITLE
Javadoc is not referenced from the documentation pages

### ISSUE THREAD

**Issue by ddebrunner** [quarks-edge/quarks.documentation#16]
Since javadoc it hte main detailed documentation it should be linked from the documentation site.

**Comment by kathyssaunders** [comment]
Updated Documentation home page with Javadoc link.  https://github.com/quarks-edge/quarks.documentation/pull/29

**Comment by ddebrunner** [comment]
Any reason the link is not under the quarks list?

**Comment by kathyssaunders** [comment]
The table of contents is generated automatically from Markdown that is available.  The Javadoc list is not in markdown, so I had to add it with a URL.  There may be a way to get it in the list, but I don't know how unless I create a Markdown document and put the link in there.  Having a separate Markdown document seemed like it just added clicks, and I thought the Javadoc is important enough that it can be first.  If you have a different opinion, we can look at other ways to do this.

**Comment by chanskw** [comment]
You can probably create a page and redirect it to the javadoc.
I haven't done it before, but here's a link on how you may be able to do  it:
http://christianspecht.de/2014/09/15/easy-meta-redirects-with-jekyll/

Another thing to try is to move the javadoc link after the include statement.  It may make it appear as part of Quarks

**Comment by kathyssaunders** [comment]
I played around with moving the Javadoc link below the include statement.  Wasn't having much luck getting the bullet to come out right, so it didn't look very good.  For now, I made a minor update, changing Javadoc to Quarks Javadoc and having it at the top.  To me, it looks  a little bit better.  When I can find the time, I'll try go look at moving it into the TOC, or someone else can try it out if they have some time.
